docs/completion.md

```diff
@@ -1,6 +1,6 @@
 <!--[metadata]>
 +++
-title = "Command Completion"
+title = "Command-line Completion"
```

Comment:
I was going by the wikipedia term for it (https://en.wikipedia.org/wiki/Command-line_completion).